net-print/brlaser/brlaser-6.2.6.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="Brother laser printer driver"
+HOMEPAGE="https://github.com/Owl-Maintain/brlaser/"
+SRC_URI="https://github.com/Owl-Maintain/brlaser/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
```

Comment:
should be `GTK-2+`, see: https://github.com/Owl-Maintain/brlaser/blob/master/src/main.cc#L5C1-L8C39

Reply:
Uh, I was looking at https://github.com/Owl-Maintain/brlaser/tree/master?tab=GPL-2.0-1-ov-file and saw it was GPL-2 Fixing right now